### DISCUSSION TITLE
including a webdav server?  /hacdias/webdav

### DISCUSSION THREAD

**Discussion by dkebler** [filebrowser/filebrowser#1760] [Ideas]
I see same dev @hacdias has a webdav server https://github.com/hacdias/webdav.  
but I don't see that as a dependency in go.mod or go.sum so looks like that is not part of filebrowser.

I could spin it up on the same root path but then i'd be maintaining two user lists, etc.

How hard for me?  the devs? would it be to incorporate this.  By the looks of the yaml config file the configuration/db structure is very similar (if not the same).

the use case is to be able to not only allow manipulation via a browser interface but also via any webdav sync client so users could be editing files/folders directly on their local machine (using their local apps) as well

**Comment by kajover** [comment]
+1. I would also like to see the functionality of having a webdav server incorporated in filebrowser.

**Comment by zhaxg** [comment]
+1 赶快来一个webdav服务，谢谢

**Comment by dkebler** [comment]
Since this isn't going to happen (any time soon) I have spun up the separate webdav server that points to the same filesystem folder that filebrowser  does and of course I must maintain via the config file the webadv user credentials.  It's not idea but it is working.
